app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -418,6 +418,12 @@ export default {
     setFormatMode(value) {
       this.updateUISettings({ display_rich_content_editor: value });
     },
+    isMessageEmpty() {
```

Comment:
Can be moved into a computed function since there are no parameters.

Reply:
Sure. I have moved the function from method to compute.